src/Options.js

```diff
@@ -156,6 +157,7 @@ addFeatureOption('generatorComprehension', EXPERIMENTAL);
 addFeatureOption('memberVariables', EXPERIMENTAL);
 addFeatureOption('require', EXPERIMENTAL);
 addFeatureOption('strongMode', EXPERIMENTAL);
+addFeatureOption('strongModeAsserts', EXPERIMENTAL);
```

Comment:
Just copying and pasting my nitty comment from earlier

Will this eventually be the flag associated with all of the dynamic aspects of strong mode? Some of these don't correspond so well to 'asserts', such as the various restrictions on strong mode functions (immutable bindings, lack of an 'arguments' object etc).

I suppose these (along with strong objects) may be difficult/impossible to implement in Traceur though.